sycl/test-e2e/bindless_images/vulkan_interop/vulkan_common.hpp

```diff
@@ -404,4 +404,120 @@ int getSemaphoreOpaqueFD(VkSemaphore semaphore) {
   return fd;
 }
 
+auto createImageMemoryBarrier(VkImage &img) {
```

Comment:
The camelCase naming here seems inconsistent with the snake_case used below. Can we be consistent in this file?